components/dataViews/TransactionInfo.tsx

```diff
@@ -64,6 +62,19 @@ const TransactionInfo = (props: Props) => {
                   </div>
                 </li>
               </>
+            ) : msg.typeUrl === "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward" ? (
+              <>
+                <li>
+                  <label>Amount:</label>
```

Comment:
This message has no amount, just those two fields: https://github.com/cosmos/cosmos-sdk/blob/v0.47.0/proto/cosmos/distribution/v1beta1/tx.proto#L66-L77